### DISCUSSION TITLE
Is there any way to edit GCP-hosted dedicated server rules (np pvp skill drain, maxplayer on server etc, cheats for admins) ?

### DISCUSSION THREAD

**Discussion by AlainRizkallah** [Nimdy/Dedicated_Valheim_Server_Script#71] [Q&A]
Noticed there is an adminlist.txt in order to add SteamIDs. I added mine but it did not allow me to use imacheater on the in-game console. (puts cheat mode to True but cheat commands do not work).

**Comment by Nimdy** [comment]
you have to restart the service after you add a admin

**Reply by KZkici** [reply]
Am I understanding this correctly that we cannot do Spawn Wood 50, on dedicated only on local or?

**Reply by Nimdy** [reply]
local only. LAter I am sure they will have something.  However, I believe I saw a mod somewhere that would allow this but you will need to run mods.

**Comment by MikeCrochet** [comment]
Yes, you can edit these settings by accessing the server configuration files via SSH. When you **[buy a dedicated server](https://arzhost.com/dedicated-servers/)** on GCP, full control allows you to customize rules like PvP, skill drain, and admin cheats.
